templates.go

```diff
@@ -1,595 +1,48 @@
x97\x1b\x03\xac\x14\x55\x55\x7c\xa5\xfa\x4f\xb0\x45\x64\xb1\x5e\x07\x80\x72\xfa\xdb\xa7\xed\xdd\x0d\x1a\xd4\x79\x1b\xe1\x52\xbd\x2f\xaf\xf5\x20\x26\xf0\xa3\x8e\x25\x54\xac\x17\x2d\x62\xb9\x67\xb1\x3c\x6f\xe5\x3f\x61\xf2\x10\xc0\x72\xd0\xba\x90\xd5\x7d\x7e\x20\xd4\xbd\x8f\xe5\xf7\x1d\x18\x0d\x0f\x10\xc8\xbc\xd7\xa6\x4d\xf6\x45\xc1\x48\xef\x35\x8a\x33\xf7\x06\xaf\xc3\x0d\xa7\x03\x72\xe3\x30\x0f\x88\x60\x50\x2e\x6c\x43\x22\x57\xdd\x75\x0d\x1c\xdd\x85\xfa\xe2\x23\x85\xf1\x5d\xfb\x6d\x2b\x7f\x54\xda\xcb\x5f\x03\x6c\xac\x37\x75\x11\x10\x60\xd8\x1e\x7a\xa4\x4d\x35\x5c\x6d\xc7\x88\x5f\x28\xf3\x71\xcc\x55\xd3\x64\x74\x47\x50\xa2\x73\xd3\x89\xbb\x03\xbd\xeb\xa2\xa7\x1e\x87\x1b\xe4\x80\x7b\x51\x2a\xa5\xbf\x7a\x50\xa9\xae\x4a\x10\x50\xa0\x91\x3e\x28\x83\xa2\xa4\xf3\xfd\xa5\xad\xd1\x97\x48\xda\x09\xf2\x1e\x0b\x91\xfb\xf6\x0e\xff\xe7\x9e\x73\xed\x7d\xfb\x8c\xa4\xd9\x95\xe7\x33\x2a\xdc\x26\x14\xb1\xce\x8f\xad\x10\xf7\xa8\xe4\x30\x39\x1b\xf4\x4e\x1b\x02\xe9\x97\xa0\x1e\x3e\x78\xdb\xca\x1f\x56\x39\x61\xc8\x15\x37\x96\x27\x42\xbc\x30\x46\xf6\x9b\x07\x2e\x45\x94\x4b\xcd\xc1\x5c\x34\x2b\x7b\x12\xeb\x96\x22\x99\x15\x8a\x9b\x5b\xcb\x92\xa1\x09\x37\x3d\xe9\xeb\xd2\x68\x6e\xe4\x38\x87\x88\x51\x21\x27\xc2\xca\xba\xd7\x59\x11\x17\xd2\xc3\xda\x52\x19\xe5\xa1\xd3\x93\x5e\x9b\x31\x1f\x53\x2c\xdd\x84\xd2\x5e\xde\xb2\x14\xe4\x1e\xbf\xb3\xd9\xd2\x36\xdb\x8a\x52\x14\xa2\xb2\x0e\xee\x88\x8f\x9c\x64\xa7\xac\x74\xbb\xa8\xb0\xd0\x27\xab\x79\xe4\x98\xea\x84\xe6\x28\x54\x8c\x30\x4e\x8c\x62\x0c\xd6\x91\xe4\x7d\x28\xed\x2e\x79\x42\xe1\xe1\x4f\x9b\x68\x5b\x77\x6f\x53\x9f\xee\x1e\x11\xd5\x56\x9b\x9b\x56\xa0\x10\xaf\x28\x9a\x55\x40\xf6\xa2\x3e\xb7\x4d\xe3\x78\x00\x9b\xf2\x74\x42\x42\x9b\x35\x24\x79\xea\x59\xee\x60\xdb\x14\x5d\xdb\x47\xdd\x6c\x0e\x4b\x87\x76\x1d\xc5\x6f\x59\xb0\x1f\xd1\x69\xca\x5b\x97\x3a\xdc\x1f\x57\xc7\xa3\xcd\x6f\x54\xb4\x07\x15\x9c\x25\x52\x5c\x42\xd2\xd2\xa8\x47\x30\x27\xac\x06\x2d\xd6\x91\x04\x4d\xba\x03\x42\x71\x44\x3a\x58\xcf\xa5\xb3\x28\x9b\x60\x40\xa0\xb4\xb1\xf2\x9c\xd4\x4e\x2c\x2c\x54\xf8\x3d\xdb\x44\xa5\x14\x3a\xe5\x96\xb1\x0b\x11\x7e\x5b\xa1\x37\xa9\xd3\x58\x70\x57\x4c\xab\x7a\x8d\x40\x1f\x43\x2e\xad\x6d\xb0\xbe\x7a\xd0\x14\x0b\x93\x77\xd4\x3d\x8b\xd2\x5c\xda\x1e\xa1\xc0\x80\xca\x76\x39\x92\x22\x7c\x43\x23\x2e\x6a\xf9\x2c\xe3\x5d\x33\xb6\x8a\xe7\xf7\x08\xaf\xb7\xf6\xbc\x2a\x81\x84\xc8\x92\x4b\x81\x04\x6d\xa6\x12\xe2\xea\xe1\xbc\x49\x96\x5c\x95\xd3\x1b\xb1\x16\x06\x55\xb2\x68\xcb\x15\xfa\x48\x2c\x55\xb0\x63\xe3\xda\x28\x32\x71\x29\x98\xb4\xf2\x55\xea\xf1\x57\x87\xd1\x81\x4c\x72\x05\x17\x73\x23\x73\xdd\x36\x4f\x1d\xf6\x3a\x1c\x39\x0b\xdc\x71\xf5\x70\xe1\xfa\xa4\x49\x57\x5e\x18\xe7\xce\xc3\x1a\xea\xee\x73\x2e\x61\x9a\x35\x4e\x61\x98\xc1\x49\x32\xd6\xf5\x87\x39\x37\xdb\xed\xb6\xe2\x82\xdd\xb2\x17\x6e\x77\xa5\x76\x6b\x70\x4b\xc0\x2e\x36\x84\xfe\xbe\x95\xe1\x88\xa5\x39\x3f\xa8\xbd\x74\x13\x37\x81\x86\xd9\x13\xb6\xbe\x0c\xd2\xc4\x59\x75\x53\x1d\xaa\xf4\x1d\x2b\xad\x92\x3b\xd5\x67\x15\xcb\x59\xc9\x48\x9f\x61\x41\x86\xa0\x34\x2c\xe6\x88\x42\xe6\x5a\xa5\xb8\x05\xed\x84\
```

Comment:
Would it be possible to make this (and similar embed variables) unexported?

Reply:
I just saw that this code in the `main` package, so it shouldn't matter for typical users. So feel free to ignore my above comment.